tools/src/main/java/org/thingsboard/client/tools/RestClient.java

```diff
@@ -71,18 +78,45 @@ public void login(String username, String password) {
         }
     }
 
-    public Device createDevice(String name) {
+    public Customer createCustomer(String title) {
+        Customer customer = new Customer();
+        customer.setTitle(title);
+        return restTemplate.postForEntity(baseURL + "/api/customer", customer, Customer.class).getBody();
+    }
+
+    public Device createDevice(String name, String type) {
         Device device = new Device();
         device.setName(name);
+        device.setType(type);
         return restTemplate.postForEntity(baseURL + "/api/device", device, Device.class).getBody();
     }
 
+    public Asset createAsset(String name, String type) {
+        Asset asset = new Asset();
+        asset.setName(name);
+        asset.setType(type);
+        return restTemplate.postForEntity(baseURL + "/api/asset", asset, Asset.class).getBody();
+    }
+
+    public Alarm createAlarm(String type, AlarmSeverity severity, AlarmStatus status, EntityId originator) {
```

Comment:
It is better to write method
 public Alarm createAlarm(Alarm alarm)

Reply:
okey!